tests/integration/rest/diddoc/query/service/negative_test.go

```diff
@@ -0,0 +1,197 @@
+//go:build integration
+
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+
+	testconstants "github.com/cheqd/did-resolver/tests/constants"
+	utils "github.com/cheqd/did-resolver/tests/integration/rest"
+	"github.com/cheqd/did-resolver/types"
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = DescribeTable("Negative: Get DIDDoc with service query", func(testCase utils.NegativeTestCase) {
+	client := resty.New()
+
+	resp, err := client.R().
+		SetHeader("Accept", testCase.ResolutionType).
+		Get(testCase.DidURL)
+	Expect(err).To(BeNil())
+	Expect(testCase.ExpectedStatusCode).To(Equal(resp.StatusCode()))
+
+	var receivedDidDereferencing types.DidResolution
+	Expect(json.Unmarshal(resp.Body(), &receivedDidDereferencing)).To(BeNil())
+
+	expectedDidDereferencing := testCase.ExpectedResult.(types.DidResolution)
+	utils.AssertDidResolution(expectedDidDereferencing, receivedDidDereferencing)
+},
+
+	Entry(
+		"cannot redirect to serviceEndpoint with not existent service query parameter",
+		utils.NegativeTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s?service=%s",
+				testconstants.SeveralVersionsDID,
+				testconstants.NotExistentService,
+			),
+			ResolutionType: testconstants.DefaultResolutionType,
+			ExpectedResult: types.DidResolution{
+				Context: "",
+				ResolutionMetadata: types.ResolutionMetadata{
+					ContentType:     types.DIDJSONLD,
+					ResolutionError: "notFound",
+					DidProperties: types.DidProperties{
+						DidString:        testconstants.SeveralVersionsDID,
+						MethodSpecificId: testconstants.SeveralVersionsDIDIdentifier,
+						Method:           testconstants.ValidMethod,
+					},
+				},
+				Did:      nil,
+				Metadata: types.ResolutionDidDocMetadata{},
+			},
+			ExpectedStatusCode: types.NotFoundHttpCode,
+		},
+	),
+
+	Entry(
+		"cannot redirect to serviceEndpoint with relativeRef query parameter",
+		utils.NegativeTestCase{
+			DidURL: fmt.Sprintf(
+				"http://localhost:8080/1.0/identifiers/%s?relativeRef=/info",
```

Comment:
Could you change `/info` to `$2Finfo` cause it requires to use URL encoding.

Reply:
Done! ✅